api/public/cerbos/request/v1/request.proto

```diff
@@ -521,6 +521,18 @@ message ListPoliciesRequest {
     (google.api.field_behavior) = OPTIONAL,
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Include disabled policies"}
   ];
+  string name_regexp = 2 [
+    (google.api.field_behavior) = OPTIONAL,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Filter policies by name with regexp"}
+  ];
+  string scope_regexp = 3 [
+    (google.api.field_behavior) = OPTIONAL,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Filter policies by scope with regexp"}
+  ];
+  string version = 4 [
```

Comment:
A performance thing primarily. It seemed reasonable to assume that `name` and `scope` would benefit from regex, but I went on the assumption that version would likely be generally short and repeatable. I was on the fence about implementing regex here too though, will happily update if we think it's worthwhile.

Reply:
I think it would be more consistent with the rest of the filters if `version` was a regex too. 

A potential use of the `version` field is to denote environments. So, I can imagine  someone might want to do a query like `version =~ (staging|qa)`